python/nano/scripts/bigdl-nano-init

```diff
@@ -215,11 +215,20 @@ if [ ! -z "${DISABLE_TCMALLOC_VAR:-}" ]; then
 	export LD_PRELOAD=`echo $LD_PRELOAD | sed "s/\s.*libtcmalloc\.so//g" | sed "s/.*libtcmalloc\.so\s*//g"`
```

Comment:
Verifying this pr looks complicated.

How about extracting the changes in `bigdl-nano-init` to another pr?